docs/source/guide/install.md

```diff
@@ -124,13 +165,14 @@ The most important change to be aware of is changes to rename "completions" to "
 If you customized the Label Studio Frontend, see the [Frontend reference guide](frontend_reference.html) for required updates to maintain compatibility with version 1.0.0.  
 
 
-
 ## Database storage
 Label Studio uses a database to store project data and configuration information. 
 
 ### SQLite database
 
-Label Studio uses SQLite by default. You don't need to configure anything. Label Studio stores all data in a single file in the specified directory of the admin user. After you [start Label Studio](start.html), the directory used is printed in the terminal. 
+Label Studio uses SQLite by default. Label Studio expects version 3.35.x or later of SQLite, so if you have another version installed, you might see unexpected behavior. 
+
+Label Studio stores all data in a single file in the specified directory of the admin user. After you [start Label Studio](start.html), the directory used is printed in the terminal. 
```

Comment:
in the specified directory of the admin user -> in the _default application data dir_ (which is different for different OS).
\+ we can also use `LABEL_STUDIO_BASE_DATA_DIR` to override this default

Reply:
updated and added ENVVAR, thanks